data/json/recipes/armor/bespoke_armor/survivor.json

```diff
@@ -49,35 +49,69 @@
     "difficulty": 7,
     "time": "12 h",
     "autolearn": true,
-    "using": [ [ "sewing_kevlar", 200 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
+    "using": [ [ "sewing_kevlar", 200 ], [ "tailoring_neoprene_patchwork", 30 ], [ "fastener_large", 1 ], [ "clasps", 8 ] ],
```

Comment:
You have removed kevlar from the list of the components, so resulting item should not be called "Kevlar" anymore or components list should be fixed (same for recipes below).